docs/BUILD.md

```diff
@@ -27,12 +27,12 @@ defaults, when possible.
 | `ROOTFS_VERSION` | Version ID of the EVE image | calculated from the git commit | `snapshot`, `1.2.3-abcd567` |
 | `ZARCH` | Hardware architecture of the resulting image | your current platform | `amd64`, `arm64`, `riscv64` |
 | `HV` | Hypervisor flavor of the resulting image | `kvm` (amd64, arm64) or `mini` (riscv64) | `kvm`, `xen`, `acrn`, `mini`, `kubevirt` |
-| `PLATFORM` | Specific platform for which to build | `generic` | `generic`, `rt`, `nvidia`, `imx8mp_pollux` |
+| `PLATFORM` | Specific platform for which to build | `generic` | `generic`, `rt`, `nvidia-jp6`, `imx8mp_pollux` |
```

Comment:
I don't think you meant to include this change in this PR.

Reply:
At least it's a part of a dedicated commit =)
For me, it's fine)

Reply:
@deitch , @OhmSpectator , while creating the documentation I've found this tiny thing to fix, I didn't plan to include this commit in this PR, but I forgot to push to another branch... since it's a very tiny thing and it has a dedicated commit, if you don't mind I would like to keep it here instead of open another PR, run checks, etc, only for this...

Reply:
I don't mind